src/commands/run.js

```diff
@@ -16,122 +15,29 @@ class RunCommand extends Command {
-            break
-          case 'error':
-            consola.debug('error', message)
-            consola.error(' Check run error', message)
-            break
-          case 'screenshot-uploads':
-            consola.debug('screenshot-uploads', message)
-            consola.info(' Screenshots:')
-
-            // Print screenshot URLs of each taken screenshot
-            message.files.forEach((file, index) => {
-              console.log(` [${chalk.bold.blue(index + 1)}]`, file.url)
-            })
-            consola.log()
-            break
-          case 'logfile':
-            consola.debug('logfile', message)
-            consola.log('')
-            consola.info(' Console Log:')
-            message.file.forEach((msg, index) => {
-              consola.log({
-                tag: msg.level.toUpperCase(),
-                message: ` [${chalk.bold.blue(index + 1)}] ${msg.msg}`,
-                time: msg.time,
-                badge: false
-              })
-            })
-
-            // Logfiles are sent last, so we can close the socket and exit at this point
-            socketClient.end()
-            process.exit(0)
-        }
-      })
-
-      // Subscribe to the 'browser-check-run' topic with this clients socketId
-      socketClient.subscribe(`browser-check-results/${socketClientId}/#`)
-
-      const { checks: parsedChecks } = await parser()
-      let selectedCheck = ''
-      if (args.checkPath) {
-        selectedCheck = Object.entries(parsedChecks).find(
-          ([name, check]) => name === args.checkPath
-        )
-      } else {
-        consola.info(
-          ' No check name passed, please search to make a selection below.\n'
-        )
-        selectedCheck = await search({
-          data: Object.entries(parsedChecks).map(([name, check]) => {
-            return { ...check, label: name }
-          }),
-          size: Object.entries(parsedChecks).length + 1,
-          cache: false,
-          debounceDelay: 200
-        })
-      }
-
-      if (!selectedCheck) {
-        consola.error(' No check found with that name, please try again.')
-        process.exit(1)
-      }
-
-      // Build check payload object
-      const browserCheck = {
-        ...selectedCheck,
-        websocketClientId: socketClientId,
-        runLocation: flags.location
-      }
+    if (!args.checkPath) {
+      const { check } = await prompt([promptCheck({ choices: Object.keys(checks).sort() })])
+      checkPath = check
+    }
 
-      // Submit check to API
-      const results = await checks.run(browserCheck)
+    const check = checks[checkPath]
 
-      if (results.status === 202) {
-        consola.info(' Check successfully submitted')
-      } else {
-        consola.error(' Error submitting check', results)
-      }
-    } catch (err) {
-      consola.error(err)
-      throw err
+    if (!check) {
+      consola.error(`Check not found, invalid check path ${checkPath}.`)
     }
+
+    check.checkType === 'BROWSER'
```

Comment:
Didn't you just create some constants for this?

Reply:
Yes 😅